src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_NUMBER = 4;
```

Comment:
Good use of a constant for the number of colors. This makes it easier to manage the number of colors and avoids magic numbers in the code.